pkgs/development/python-modules/marqo/default.nix

```
@@ -0,0 +1,92 @@
+{
+  lib,
+  python3,
+  fetchFromGitHub,
+}:
+
+let
+  py = python3 // {
+    pkgs = python3.pkgs.overrideScope (
+      final: prev: {
+        urllib3 = prev.urllib3.overridePythonAttrs (prev: rec {
+          pyproject = true;
+          nativeBuildInputs = [ final.setuptools ];
+          version = "1.26.19";
+          src = prev.src.override {
+            inherit version;
+            hash = "sha256-Pj11OoYYuG194zO0IjAF9ocgvNan0ry5+9IinsfB5Ck=";
+          };
+        });
+      }
+    );
+  };
+in
+py.pkgs.buildPythonPackage rec {
+  name = "marqo";
+  version = "3.7.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "marqo-ai";
+    repo = "py-marqo";
+    rev = "refs/tags/${version}";
+    hash = "sha256-qwGSw6NJJzay1hvSKIkV7ONlrxut196j2pQRCWYI3YI=";
+  };
+
```

```suggestion

  pythonRelaxDeps = [
    "urllib3"
  ];
```
Package does not use urllib3 directly. Our other python packages work with urllib3 version 2, I think.

---

In such a case, we can use `pythonRemoveDeps` and remove `urllib3` from dependencies.